### PR TITLE
[WIP] Adds Space Jesus the Messiah

### PR DESCRIPTION
**Pull request by Kierany9**
### This is a work in progress, I'm waiting on the sprites, some of the spells need polish, one of them isn't even done and there are a couple of additions I want to make.

Are you tired of everybody fucking dying in zero seconds flat? Is there an emergency that doesn't need a cleanup crew but rather the good word of the lord? Worry no more, Space Jesus* is here!

The Messiah can be spawned in somewhere in maint at any time by admins and comes equipped with six special spells known as Miracles. The Messiah will be unable to use guns nor handcuffs and cannot take off his robe nor wear armour (Once I get around to it).

- **Resurrection:** Simply brings a target back from the dead.
- **Blood to Wine:** Transforms the blood of anybody nearby into alcohol causing stupor and potential paralysis. Potency varies by distance.
- **Cure the Sick:** Heals a target back to full health, removing any mutations, disease or sickness affecting the target.
- **Parting Waves:** Once used to split the ocean in two, this miracle is now relegated to opening ~your mom's legs~ airlocks. Extremely useful. Reskin of Knock.
- **Ticket to Ride:** The Messiah blesses the target with extreme luck, causing many incoming attacks to simply bounce off. However, the negative fortune must go elsewhere, and the messiah takes the brunt of it himself in the form of clone damage.
- **Repent for your Sins:** The most powerful miracle of all, after a charge time of 2 minutes that requires the target to be completely still, Jesus can remove somebody's antagonist status.

*Name varies depending on chaplain, this may be Nalar if the chaplain is a coder or ChronIsACuck if he is a five year old

### CODE REVIEW

**Comment by banthebantz** [comment]
what

**Comment by Kierany9** [comment]
Repent

**Comment by tortellinitony** [comment]
why would this ever get spawned though

**Comment by gumner** [comment]
murderboning fags?

**Comment by Anderbergst** [comment]
new spell - rise from the dead

after 3 minutes, you rise from dead even if you are gibbed and etc

**Comment by MethMedic** [comment]
Yea I could see this for being a counter to murderboners. Although Repent is way too OP as you can just stand in maint.

**Comment by yobob591** [comment]
that requires the target to stand still too @MethMedic

**Comment by SpankMasterson** [comment]
make it a random event maybe, either way i like +1

**Comment by tortellinitony** [comment]
get rid of repent and make big cooldowns for the other spells, and it would be tame enough for an event

**Comment by blobbernaut** [comment]
+2

**Comment by blobbernaut** [comment]
Also, add 'in progress' tag

**Comment by McBawbaggings** [comment]
>He implemented Ticket To Ride

+1 _amazing_ pr.

File: code/game/gamemodes/messiah/miracles.dm [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+	panel = "Miracles"
+	charge_max = 6000
+	invocation_type = "none"
+
+/obj/effect/proc_holder/spell/targeted/jesus_deconvert/cast(list/targets,mob/user = usr)
+
+	for(var/mob/living/carbon/human/target in targets)
+		if(target.stat != DEAD || target.mind)
+			var/is_antag = target.mind.special_role
+			if(do_mob(user,target,400))
```
Comment (McBawbaggings) [inline code/game/gamemodes/messiah/miracles.dm]: Won't this just silently fail if you use it on a good guy?

**Comment by Sarcalogo** [comment]
Do you really have to taint Jesus Christ of all people with stupid JoJo shit? Will you JoJofags ever fuck off? I'm those close to wiping all JoJofags off of the face of the earth. Nuclear bombs.

**Comment by Kierany9** [comment]
>He doesn't know that Jesus is an actual jojo character
>Literally the only thing vaguely relating to jojo in this PR is one of the spells

go away sarcalogo

**Comment by FuryMcFlurry** [comment]
spell icons WIP
http://image.prntscr.com/image/6105859f554a4c3fae54dc482ea568e9.png

**Comment by Dudeinyourbackyard** [comment]
he actually did it
jesus really is real
+1

**Comment by Swagile** [comment]
removing antag status? nah, easier to just kill the dude if sec catches a antag

and you won't be able to do shit to the dude by yourself to make repent worth it

replace the spell with a revive despite gibbing / cremate and this is a good gimmick protagonist

**Comment by gumner** [comment]
there should be a way for the chaplain to summon him tbh

**Comment by Sarcalogo** [comment]
No, the corpse of Jesus is a plot device in JoJo, Jesus Christ himself is not a character. You're just shoe horning in JoJo shit (ticket to ride) where it isn't needed.

**Comment by Sarcalogo** [comment]
And Ticket to Ride has nothing to do with Jesus Christ as he is represented in the Bible, don't be a fucking faggot.

**Comment by ArcLumin** [comment]
And being on a space station has nothing to do with Jesus Christ as he is represented in the Bible, don't be a fucking faggot.

**Comment by ArcLumin** [comment]
wow jesus can open any door where'd the bible say that immershun breaking go kys faggot @Kierany9

**Comment by Pyko1** [comment]
Should be able to spawn more fish, if he has one to begin with, to feed the station.

**Comment by Pyko1** [comment]
@ArcLumin I believe its an artistic freedom based on John 20:19

**Comment by Sarcalogo** [comment]
You'll defend any decision so long as one of your friends has made it, may as well rename HippieStation JoJoStation at this point. 

All other abilities are loosely based on the mythos of Christ, some retarded shit from JoJo that is loosely associated with Jesus (Funny Valentine ---> Corpse Parts ----> Jesus) is fucking cancerous JoJo shit not part of the Christ character. When there is already 1000000 JoJo references, and you keep adding more and more, you're like Bronies at this point, except with JoJo. You had to desecrate Jesus of all people with your JoJo crap.

**Comment by blobbernaut** [comment]
Sarc is sperging the fuck out, cancel the PR everybody

**Comment by Sarcalogo** [comment]
Nothing will ever happen. May as well make a rule mandating for everything you add, one JoJo related thing must be added. JoJofags, I am currently developing a bioweapon to wipe you off of the face of the Earth, be vigilant. Your deaths come soon.

**Comment by ArcLumin** [comment]
I don't really care about this PR much, don't support it but don't much care either way. I just found your comment especially hilarious considering that you didn't find it at all absurd of people playing jesus on a space simulator but suddenly were completely unnutted as soon as someone referenced something you didn't like.

As for the JoJoshit, we're just adding code we think players will enjoy, and lots of the players enjoy jojo. If you don't like the references it's an easy reskin or rename, just give it a new pull request to change the name to reduce direct references.

**Comment by blobbernaut** [comment]
Alternatively, fuck off, that also works

**Comment by Sarcalogo** [comment]
I am fueled by my hatred of JoJofags, can I really undo them..? What happens when they all vanish? What will be my motivation to get up in the day? What's the point of living without JoJofags to hate, exterminate, and humiliate? What is the POINT of living?

**Comment by ArcLumin** [comment]
I guess you could say you really can't STAND the thought of a world without JoJofags

**Comment by ArcLumin** [comment]
>blood to wine doesn't convert blood in the person's system to wine

**Comment by Kierany9** [comment]
>Jesus Christ himself is not a character

[yes](http://vignette4.wikia.nocookie.net/jjba/images/6/6c/Jesus.PNG/revision/latest?cb=20150113171350) [he](http://vignette3.wikia.nocookie.net/jjba/images/a/ad/Jesus_advice.png/revision/latest/top-crop/width/480/height/480?cb=20160506132256) [is](http://vignette2.wikia.nocookie.net/jjba/images/9/96/Blackmore_Jesus.JPG/revision/latest/top-crop/width/480/height/480?cb=20150113171537)

also ticket to ride is literally the power his corpse grants
also also i had the idea of having a spawnable space jesus long before becoming a jojo fan
also also also ticket to ride is the spell that hasn't even been added yet :^)

>I believe its an artistic freedom based on John 20:19

Actually, I based it on [Exodus 14:21](http://biblehub.com/exodus/14-21.htm). Jesus isn't Moses but then again, this isn't Jesus, this is _*Space Jesus*_, or more accurately, whatever the Chaplain decided to name their deity for the round.

**Comment by Pyko1** [comment]
I was talking about opening a door what arc was talking about. hence [John 20:19](http://biblehub.com/john/20-19.htm)

**Comment by Kierany9** [comment]
>opening the door for jews

big mistake jesus

**Comment by ArcLumin** [comment]
if my deity is badmin would he be badmin jesus?

**Comment by Kierany9** [comment]
he would be badmin

**Comment by Spacedong** [comment]
remove the jojo reference crap and its good

**Pull request closed without merging**